decidim-core/app/views/decidim/account/_account_notification.js.erb

```diff
@@ -0,0 +1,15 @@
+if (typeof accountNotification === 'undefined') {
+  var accountNotification = document.querySelector(".account-notification");
+}
+
+if (typeof messageDiv === 'undefined') {
+  var messageDiv = document.createElement("div");
+  messageDiv.classList.add("account-message")
+}
+
+accountNotification.appendChild(messageDiv);
+messageDiv.innerHTML = "<%= text %>";
+
+accountNotification.classList.add("<%= alert_class %>")
+accountNotification.classList.remove("hide");
+accountNotification.setAttribute("style", "display: flex;");
```

Comment:
Do we need to explicitly set the inline styling here? I tried disabling this with inspect element and it seems to work fine at least in Chrome.

Reply:
If you do following:
1. click "Send again"
2. Close notification
3. click "Cancel"
You wouldn't see notification anymore because closing notification adds "display: none;"

Reply:
Ah OK. Maybe then just `accountNotification.removeAttribute("style")`?

Reply:
Then it will look like this:

![image](https://user-images.githubusercontent.com/19709320/162179744-da43ca14-53cc-432e-be87-6d1033dd4c61.png)


Compared to flex:

![image](https://user-images.githubusercontent.com/19709320/162179669-8d2c7567-d5d3-455a-8279-07bfd52cd4f3.png)

Reply:
Moved styles to user.scss so we don't have to use inline css.

Reply:
I was still thinking about this and could we instead utilize the "normal" notification position for these messages that we are also using when you click the "Update account" button:
![Notifications](https://user-images.githubusercontent.com/864340/162448335-b1e6a7f6-15e2-4c60-9250-26001d6e6989.png)

Or is there some particular reason why we want to show the notification inside the account container?

Reply:
I think that would require reloading page after these links and that would dismiss all the other changes made to the form(?)

Reply:
I moved it there but then there can be two alerts

![image](https://user-images.githubusercontent.com/19709320/163000768-e596df3d-f3a6-4fa4-bd0d-7cafbf161596.png)

Reply:
> I moved it there but then there can be two alerts

Yep I though if we could check if there is already an alert and replace the previous alert in case there was one?

> And maybe check if there is already an alert, then replace it?

I think this works better that the alert goes where it normally appears.

Reply:
Sorry I didn't read your previous message properly, now it hides existing notification.